hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -16,6 +17,45 @@
 log = logging.getLogger(__name__)
 
 
+def get_coords(in_mli_par, ref_azlin, ref_rpix, in_dem_par=None):
+    """
+    inputs: mil.par, dempar, reference point  in SAR space (ref_azlin, ref_rpix)
+    returns: coords={"row_s":row_s,"col_s":col_s,"row_m":row_m,"col_m":col_m,"x":x,"y":y,"lat":lat,"lon":lon}
+    """
+    def _coord_lst(cmd):
+        coord_txt = subprocess.run(cmd, capture_output=True, text=True)
+        lst = coord_txt.stdout.split('\n')
+        coord_str = [s for s in lst if "selected" in s]
+        coord_lst = ' '.join(coord_str[0].split()).split(" ")[: -1]
+        coord_lst = [float(s) for s in coord_lst]
+        return coord_lst
```

Comment:
I think it'd be better to move this function outside of `get_coords` and give it a more descriptive name. 

Really, this is `coords_from_sarpix_coord`, and could have the command template inside where you provide the command arguments as parameters